### ISSUE TITLE
Dockerhub image tags outdated, fails

### ISSUE THREAD

**Issue by mikecook** [hadolint/hadolint#156]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Dockerhub latest tag matches latest Github tag
Dockerhub tags align with Github tags
Automated builds configured for Dockerhub: https://docs.docker.com/docker-hub/builds/

### Actual behavior/ Steps to reproduce
```
$ docker pull lukasmartinelli/hadolint:latest
latest: Pulling from lukasmartinelli/hadolint
Digest: sha256:fc4d30390f090f055bde0ca5aa6b72a05aa000cb8f385ba195c9fff73210db61
Status: Image is up to date for lukasmartinelli/hadolint:latest
$ docker run --rm -i lukasmartinelli/hadolint hadolint --version
Haskell Dockerfile Linter v1.2.2-65-g94e9c5d
$ docker run --rm -i lukasmartinelli/hadolint hadolint - < Dockerfile
/dev/stdin:3:13 unexpected '/' expecting space, "\t", "ONBUILD", "FROM", "COPY", "RUN", "WORKDIR", "ENTRYPOINT", "VOLUME", "EXPOSE", "ENV", "ARG", "USER", "LABEL", "STOPSIGNAL", "CMD", "SHELL", "MAINTAINER", "ADD", "#", "HEALTHCHECK" or end of input
```
```
$ hadolint --version
Haskell Dockerfile Linter 1.2.6-no-git
$ hadolint Dockerfile 
$ 
```

**Dockerfile** (if relevant)

```Dockerfile
FROM alpine:3.7
EXPOSE 53 53/UDP
```

**Additional environment details (OS, stack version, etc.)**
Tested on OSX and Linux

**Comment by mikecook** [comment]
I see it's now hadolint/hadolint. Closing.
